OpenApi document generation is broken

### Is there an existing issue for this?

- [x] I have searched the existing issues

### Describe the bug

When publishing the `TodosApi` benchmark in https://github.com/aspnet/Benchmarks/tree/main/src/BenchmarksApps/TodosApi I'm getting the following build error:

```
❯ \git\aspnetcore\.dotnet\dotnet.exe publish TodosApi.csproj -c Release --disable-build-servers /p:MicrosoftNETCoreAppPackageVersion=10.0.0-alpha.1.25072.13 /p:MicrosoftAspNetCoreAppPackageVersion=10.0.0-alpha.2.25073.4 /p:GenerateErrorForMissingTargetingPacks=false /p:RestoreNoCache=true /p:MicrosoftWindowsDesktopAppPackageVersion=10.0.0-alpha.1.25073.1 /p:MicrosoftNETPlatformLibrary=Microsoft.NETCore.App /p:PublishAot=false /p:EnableRequestDelegateGenerator=false --framework net10.0 --self-contained -r win-x64 /bl
You are using a preview version of .NET. See: https://aka.ms/dotnet-support-policy
  TodosApi net10.0 failed with 2 error(s) (0.6s) → bin\Release\net10.0\win-x64\TodosApi.dll
    D:\packages\NuGet\cache\microsoft.extensions.apidescription.server\10.0.0-alpha.2.25073.4\build\Microsoft.Extensions.ApiDescription.Server.targets(68,5): error : Could not execute because the specified command or file was not found.
    D:\packages\NuGet\cache\microsoft.extensions.apidescription.server\10.0.0-alpha.2.25073.4\build\Microsoft.Extensions.ApiDescription.Server.targets(68,5): error MSB3073: The command "dotnet "D:\packages\NuGet\cache\microsoft.extensions.apidescription.server\10.0.0-alpha.2.25073.4\build\../tools/dotnet-getdocument.dll" --assembly "D:\git\Benchmarks\src\BenchmarksApps\TodosApi\bin\Release\net10.0\win-x64\TodosApi.dll" --file-list "obj\TodosApi.OpenApiFiles.cache" --framework ".NETCoreApp,Version=v10.0" --output "D:\git\Benchmarks\src\BenchmarksApps\TodosApi" --project "TodosApi" --assets-file "D:\git\Benchmarks\src\BenchmarksApps\TodosApi\obj\project.assets.json" --platform "x64" --runtime "win-x64 --self-contained" " exited with code 1.
```

The problem appears to be that the `dotnet-getdocument.dll` file is not found in the `microsoft.extensions.apidescription.server` nuget package under `tools` directory. This may have been caused by https://github.com/dotnet/aspnetcore/commit/d4880ed4160b476346850ac653ce4b829ab09c94.

cc @captainsafia 

### Expected Behavior

The TodosApi project should publish successfully.

### Steps To Reproduce

1. Clone and `cd` into https://github.com/aspnet/Benchmarks/tree/main/src/BenchmarksApps/TodosApi
2. Change the `TargetFramework` of the project to `net10.0`
3. Get the latest .NET 10 SDK daily build
4. Run the following command:

```sh
dotnet.exe publish TodosApi.csproj -c Release --disable-build-servers /p:MicrosoftNETCoreAppPackageVersion=10.0.0-alpha.1.25072.13 /p:MicrosoftAspNetCoreAppPackageVersion=10.0.0-alpha.2.25073.4 /p:GenerateErrorForMissingTargetingPacks=false /p:RestoreNoCache=true /p:MicrosoftWindowsDesktopAppPackageVersion=10.0.0-alpha.1.25073.1 /p:MicrosoftNETPlatformLibrary=Microsoft.NETCore.App /p:PublishAot=false /p:EnableRequestDelegateGenerator=false --framework net10.0 --self-contained -r win-x64 /bl
```

### Exceptions (if any)

_No response_

### .NET Version

_No response_

### Anything else?

Here is a binlog of the build error:

[msbuild.binlog.zip](https://github.com/user-attachments/files/18540531/msbuild.binlog.zip)

--- a/src/Tools/Extensions.ApiDescription.Server/src/Microsoft.Extensions.ApiDescription.Server.nuspec
+++ b/src/Tools/Extensions.ApiDescription.Server/src/Microsoft.Extensions.ApiDescription.Server.nuspec
@@ -8,6 +8,7 @@
     $CommonFileElements$
     <file src="build\*" target="build" />
     <file src="buildMultiTargeting\*" target="buildMultiTargeting" />
+    <file src="$artifactsBinDir$\dotnet-getdocument\$configuration$\net10.0\publish\*.*" target="tools" />
     <file src="$artifactsBinDir$\GetDocument.Insider\$configuration$\net462\*.*" target="tools\net462" />
     <file src="$artifactsBinDir$\GetDocument.Insider\x86\$configuration$\net462\*.*" target="tools\net462-x86" />
     <file src="$artifactsBinDir$\GetDocument.Insider\$configuration$\net10.0\publish\*.*" target="tools\net10.0" />

